src/util/logging.ts

```diff
@@ -0,0 +1,16 @@
+import { Express, NextFunction, Request, Response } from "express";
+import { ValidateError } from "@tsoa/runtime";
+
+export function addTsoaValidationFailureLogging(app: Express) {
+  app.use(
+    (err: unknown, _req: Request, _res: Response, next: NextFunction): void => {
+      if (err instanceof ValidateError) {
+        console.error(
```

Comment:
May want to consider middleware like this that automatically adds the appropriate status code to the response actually. Nice.

Reply:
Yeah that could be cool, haven't been busy with the status codes yet.